editor/src/components/canvas/canvas.ts

```diff
@@ -327,13 +361,13 @@ const Canvas = {
               height: frameWithPath.frame.height === 0 ? 1 : frameWithPath.frame.height,
             }),
             canvasRectangle({
-              x: canvasPosition.x - looseReparentThreshold,
-              y: canvasPosition.y - looseReparentThreshold,
-              width: 2 * looseReparentThreshold,
-              height: 2 * looseReparentThreshold,
+              x: canvasArea.x - looseReparentThreshold,
+              y: canvasArea.y - looseReparentThreshold,
+              width: canvasArea.width * 2 * looseReparentThreshold,
```

Comment:
this looks wrong. originally the ` 2 * looseReparentThreshold` is a fixed value, 10 pixels. this will now be area width times 10, which is MASSIVE for a non-zero selection area! 

```suggestion
              width: canvasArea.width + 2 * looseReparentThreshold,
```